packages/api-main/package.json

```
@@ -1,21 +1,20 @@
 {
-  "name": "api-feed",
+  "name": "@dither.chat/api-main",
```

Is there a reason to keep using the "main" suffix?

Wouldn't it be more semantic to just use:
```suggestion
  "name": "@dither.chat/api",
```